src/AdminService/Startup.cs

```diff
@@ -1,265 +1,267 @@
+
+      services.AddBusinessObjects();
+
+      ConfigureMassTransit(services);
+    }
+
+    public void Configure(IApplicationBuilder app, ILoggerFactory loggerFactory)
+    {
+      UpdateDatabase(app);
+
+      app.UseForwardedHeaders();
+
+      app.UseExceptionsHandler(loggerFactory);
+
+      app.UseApiInformation();
+
+      app.UseRouting();
+
+      app.UseMiddleware<TokenMiddleware>();
+
+      app.UseCors(CorsPolicyName);
+
+      app.UseEndpoints(endpoints =>
+      {
+        endpoints.MapControllers().RequireCors(CorsPolicyName);
+
+        endpoints.MapHealthChecks($"/{_serviceInfoConfig.Id}/hc", new HealthCheckOptions
+        {
+          ResultStatusCodes = new Dictionary<HealthStatus, int>
+          {
+            { HealthStatus.Unhealthy, 200 },
+            { HealthStatus.Healthy, 200 },
+            { HealthStatus.Degraded, 200 },
+          },
+          Predicate = check => check.Name != "masstransit-bus",
+          ResponseWriter = UIResponseWriter.WriteHealthCheckUIResponse
+        });
+      });
+    }
+
+    #endregion
+
+    #region private methods
+
+    private (string username, string password) GetRabbitMqCredentials()
+    {
+      static string GetString(string envVar, string formAppsettings, string generated, string fieldName)
+      {
+        string str = Environment.GetEnvironmentVariable(envVar);
+        if (string.IsNullOrEmpty(str))
+        {
+          str = formAppsettings ?? generated;
+
+          Log.Information(
+            formAppsettings == null
+              ? $"Default RabbitMq {fieldName} was used."
+              : $"RabbitMq {fieldName} from appsetings.json was used.");
+        }
+        else
+        {
+          Log.Information($"RabbitMq {fieldName} from environment was used.");
+        }
+
+        return str;
+      }
+
+      return (GetString("RabbitMqUsername", _rabbitMqConfig.Username, $"{_serviceInfoConfig.Name}_{_serviceInfoConfig.Id}", "Username"),
+        GetString("RabbitMqPassword", _rabbitMqConfig.Password, _serviceInfoConfig.Id, "Password"));
+    }
+
+    private void ConfigureMassTransit(IServiceCollection services)
+    {
+      (string username, string password) = GetRabbitMqCredentials();
+
+      services.AddMassTransit(x =>
+      {
+
+        x.UsingRabbitMq((context, cfg) =>
+        {
+          cfg.Host(_rabbitMqConfig.Host, "/", host =>
+          {
+            host.Username(username);
+            host.Password(password);
+          });
+
+        });
+
+        x.AddRequestClients(_rabbitMqConfig);
+      });
+
+      services.AddMassTransitHostedService();
+    }
+
+
+    private void UpdateDatabase(IApplicationBuilder app)
+    {
+      using var serviceScope = app.ApplicationServices
+        .GetRequiredService<IServiceScopeFactory>()
+        .CreateScope();
+
+      using var context = serviceScope.ServiceProvider.GetService<AdminServiceDbContext>();
+
+      context.Database.Migrate();
+    }
+
+    private string HidePassord(string line)

```

Comment:
remove this method and use HidePassword from Kernel.Helpers like Spartak and Mariya said